### PR TITLE
Add development instructions for SAPPHiRE

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
Based on Gitter chat witih @tomkooij over on [Open Cosmics](https://github.com/OpenCosmics/opencosmics.github.io).

Let me know if you want me to edit this before you merge, if you want to merge it, i.e. :)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,6 +57,18 @@ To check if it worked start Python and load the package:
...
+Development
+-----------
+
+First, install anaconda and run
+
+    $ conda install numpy scipy pytables
+
+After this, SAPPHiRE should work out of the box:
+
+    $ git clone ... && cd sapphire && python setup.py develop
```
Comment (153957) [inline README.rst]: You should make it easy to copy-paste.
So add the git url to the clone command: `https://github.com/HiSPARC/sapphire.git`
And anaconda is not really needed, miniconda also suffices, you could add a link to that installer:
http://conda.pydata.org/docs/install/quick.html

Comment (RaoOfPhysics) [inline README.rst]: Will do.

**Comment by RaoOfPhysics** [comment]
@153957: Updated and amended the commit. I'm not familiar with `.rst`, so please let me know if this is ok.

**Comment by 153957** [comment]
You can see your rendered readme here: https://github.com/RaoOfPhysics/sapphire/tree/update-readme#development
The lines before the code should both end in double colons (`::`) to indicate a code section is coming.
Please create new commits, do not amend/force push to prevent possible issues with other peoples clones, and keep the history clearer.

**Comment by RaoOfPhysics** [comment]
@153957: Sounds good. Thank you very much. :)

File: README.rst [hunk 2]
```diff
@@ -60,11 +60,11 @@ You're done!
 Development
 -----------
 
-First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run
+First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run::
 
     $ conda install numpy scipy pytables
 
-After this, SAPPHiRE should work out of the box:
+After this, SAPPHiRE should work out of the box:::
```
Comment (153957) [inline README.rst]: Only two, not three colons ;)

Comment (RaoOfPhysics) [inline README.rst]: Yeah, I wanted one of the colons to be, you know, a colon. :P Oh, well. Making one more change then. :)

Comment (153957) [inline README.rst]: Two colons do produce one colon, as can be seen in the Installation section.
If you want no colons you can use some code-block tag in reStructuredText.

**Pull request merged**